src/renderer/components/Editor/ProjectTooltip.tsx

```diff
@@ -0,0 +1,21 @@
+import { styled, Tooltip, tooltipClasses, TooltipProps } from '@mui/material';
+import colors from 'renderer/colors';
+
+const ProjectTooltip = styled(({ className, ...props }: TooltipProps) => (
```

Comment:
I think there's too much going on inline here, can we separate out into a couple of steps so it's clearer what's going on? Might be able to get a definitive type here as well if that's still an issue

Reply:
Yea, I just copy pasted this from the mui examples, so I'm not 100% clear on it either. I have separated it out a bit I think it's clearer now